src/Microsoft.OData.Client/Materialization/ODataDeletedMaterializerEntry.cs

```diff
@@ -0,0 +1,52 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataDeletedMaterializerEntry.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Diagnostics;
+
+namespace Microsoft.OData.Client.Materialization
+{
+    /// <summary>
+    /// Materializer state for a given <see cref="ODataDeletedResource"/>.
+    /// </summary>
+    internal class ODataDeletedMaterializerEntry : IMaterializerState

```

Comment:
Why is this named differently from the other states which don't have the `OData` prefix?

Reply:
changed it to `MaterializerDeletedEntry`